api/webhook/post.go

```diff
@@ -381,6 +397,14 @@ func PostWebhook(c *gin.Context) {
 
 					return
 				}
+
+				l.WithFields(logrus.Fields{
+					"deployment":    dr.GetNumber(),
+					"deployment_id": dr.GetID(),
+					"org":           repo.GetOrg(),
+					"repo":          repo.GetName(),
+					"repo_id":       repo.GetID(),
+				}).Info("deployment created in database")
```

Comment:
same question, redundant language

Reply:
should be addressed in https://github.com/go-vela/server/pull/1145/commits/53edfd650dbff8ab902ed3490e4fc55bd36ee151